boards/lora-e5-dev/include/board.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2021 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_lora-e5-dev
+ * @{
+ *
+ * @file
+ * @brief       Pin definitions and board configuration options for
+ *              LoRa-E5 Development Board
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.yyfr>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "kernel_defines.h"
+#if IS_USED(MODULE_SX126X_STM32WL)
+#include "sx126x.h"
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Sub-GHz radio (LoRa) configuration
+ * @{
+ */
+#define SX126X_PARAM_SPI                    (SPI_DEV(0))
+#if IS_USED(MODULE_SX126X_STM32WL)
+extern void lora_e5_dev_sx126x_set_rf_mode(sx126x_t *dev, sx126x_rf_mode_t rf_mode);
+#define SX126X_PARAM_SET_RF_MODE_CB         lora_e5_dev_sx126x_set_rf_mode
+#define SX126X_PARAM_TYPE                   SX126X_TYPE_STM32WL
+#endif
```

Comment:
If the radio module is part of the CPU the config should probably be moved there as it's not board specific.

Reply:
Hmmm I'm not sure if its CPU specific, the lora-e5 does not have the same RF modes as the nucleo-wl55jc, I'm not sure
if this is BOARD or CPU_MODEL specific.